tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -103,44 +103,42 @@ function SplashPage() {
               fulfilled their reporting requirements.
             </p>
             <Button
-              className="width-full sign-in-button"
+              className="width-full"
               type="button"
               id="loginDotGovSignIn"
               onClick={signInWithLoginDotGov}
             >
-              <div className="mobile:margin-x-auto mobile-lg:margin-0">
+              <div>
                 Sign in with
                 <img
-                  className="mobile:margin-x-auto mobile:padding-top-1 mobile-lg:margin-0 mobile-lg:padding-top-0 width-15 padding-left-1"
+                  className="mobile:margin-x-auto mobile:padding-top-1 mobile:margin-0 mobile:padding-top-0 width-15 padding-left-1"
                   src={loginLogo}
                   alt="Login.gov"
                 />
-                for grantees
+                &nbsp; for grantees
               </div>
               <span className="visually-hidden">Opens in a new website</span>
             </Button>
             <Button
-              className="width-full sign-in-button margin-top-3"
+              className="width-full margin-top-3"
               type="button"
               id="acfAmsSignIn"
               onClick={signInWithAMS}
             >
-              <div className="mobile:margin-x-auto mobile-lg:margin-0">
-                Sign in with ACF AMS for ACF staff
-              </div>
+              <div>Sign in with ACF AMS for ACF staff</div>
               <span className="visually-hidden">Opens in a new website</span>
             </Button>
           </div>
         </div>
       </section>
 
-      <section className="grid-container usa-section">
-        <div className="grid-row grid-gap">
-          <div className="tablet:grid-col-4">
-            <h2 className="resources-header font-heading-2xl margin-top-0 tablet:margin-bottom-0">
+      <section className="desktop:grid-container-widescreen padding-top-4 usa-section">
+        <div className="grid-row desktop:margin-left-0 desktop:margin-right-0 mobile:margin-left-2 mobile:margin-right-2">
```

Comment:
By setting the margins for both desktop and mobile the spacing of the grid-container content is as it should be. Since there was only mobile styling before, desktop views were also using those margins, which accounted for the content not being centered.

Reply:
This could be put into the main style sheet as a new class.